iOSOverUsbSupport.cs

```diff
@@ -159,6 +160,9 @@ class PosixDllLoader: IDllLoader
 
 		public IntPtr LoadLibrary(string fileName)
 		{
+			if (!File.Exists (fileName))
+				throw new FileNotFoundException (fileName);
```

Comment:
I don't think this makes sense in the general case - dlopen uses the system library lookup mechanism.
Are we always passing an actual path name here?